src/extra/libs/rlottie/lv_rlottie.h

```diff
@@ -33,9 +31,11 @@ typedef enum {
     LV_RLOTTIE_CTRL_LOOP     = 8,
 } lv_rlottie_ctrl_t;
 
+/** definition in lottieanimation_capi.c */
+struct Lottie_Animation_S;
 typedef struct {
     lv_img_t img_ext;
-    Lottie_Animation * animation;
+    struct Lottie_Animation_S * animation;
```

Comment:
You need to forward declare the structure to use a pointer to it. You can add a `struct Lottie_Animation;` above this declaration. Else the compiler will error here.